ksqldb-functional-tests/src/test/resources/historical_plans/joins_-_table_to_table_wrapped_single_field_value_schema_on_inputs/5.5.0_1581572097364/spec.json

```diff
@@ -103,6 +103,14 @@
         "ID2" : null
       },
       "timestamp" : 30
+    }, {
+      "topic" : "OUTPUT",
+      "key" : "0",
+      "value" : {
+        "ID1" : null,
+        "ID2" : null
+      },
+      "timestamp" : 40
```

Comment:
@vcrfxia @agavra -- for my own education: why is this output (and the previous one with ts=30) `<null, null>` and not just `null` as for ts=50 below.

We are testing an inner join here so I though the value should be tombstone here?

Reply:
The input record is not a tombstone: https://github.com/confluentinc/ksql/blob/c2ace3cb4ad05b03796b007350efc26b1846ef62/ksqldb-functional-tests/src/test/resources/query-validation-tests/joins.json#L1391
ksqlDB distinguishes between "null value" and "object containing null value".

Reply:
Ah. that makes sense. Missed this "detail". Thanks for the pointer.